Example/Sources/View Controller/MerchantAssetsViewController.swift

```diff
@@ -9,54 +9,138 @@
 import PrimerSDK
 import UIKit
 
+fileprivate enum PrimerPaymentMethodTypeForAssets: String, CaseIterable, Equatable, Hashable {
```

Comment:
This is to allow iteration over the various assets as this type is internal for the SDK.